docker/base-deps/Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN sudo apt-get update -y && sudo apt-get upgrade -y \
         openssh-client \
         gnupg; fi) \
     && if [[ "${PYTHON_VERSION}" = "3.11" || "${PYTHON_VERSION}" =~ ^3\.11\. ]]; then \
-            MINICONDA_VERSION="py310_23.3.1-0"; \
+            MINICONDA_VERSION="py311_23.10.0-1"; \
```

Comment:
i am surprised that a miniconda of py310 actually worked with python 3.11.

Reply:
someone mentioned on the conda github issue that it doesn't guarantee to work which be why customers run into issues